ui/src/utils/ajax.js

```diff
@@ -36,46 +42,45 @@ const generateResponseWithLinks = (response, newLinks) => {
 
 const AJAX = async (
   {url, resource, id, method = 'GET', data = {}, params = {}, headers = {}},
-  excludeBasepath
+  {excludeBasepath} = {}
```

Comment:
excludeBasepath is a bool when it is set, should the default be a bool too?

Reply:
see above